src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ProductVariant.xml

```diff
@@ -48,6 +42,12 @@
                 <attribute name="path">/admin/product-variants</attribute>
             </collectionOperation>
 
+            <collectionOperation name="admin_post">
+                <attribute name="method">POST</attribute>
+                <attribute name="path">/admin/product-variants</attribute>
+                <attribute name="controller">api_platform.doctrine.orm.data_persister</attribute>
```

Comment:
Is this line needed?

Reply:
AFAIK, no. Can we just remove this config?

Reply:
Removed :)